src/cache/index.ts

```diff
@@ -112,45 +110,24 @@ export const calculateFeedId = <T extends EndpointGenerics>(
  *
  * @example
  * ```
- * calculateKey({ base: 'ETH', quote: 'BTC' }, ['base','quote'])
- * // equals `|base:eth|quote:btc`
+ * calculateKey({ base: 'ETH', quote: 'BTC' })
+ * // equals `{"base":"eth","quote":"btc"}`
  * ```
  */
 export const calculateKey = <CustomSettings extends SettingsMap>(
   data: unknown,
-  paramNames: string[],
   adapterConfig: AdapterConfig<CustomSettings>,
 ): string => {
```

Comment:
Need to update the JSDoc for this function. Seems like it was incomplete to begin with